package.json

```diff
@@ -72,7 +72,7 @@
     "jquery-i18next": "1.2.1",
     "js-md5": "0.6.1",
     "jwt-decode": "2.2.0",
-    "lib-jitsi-meet": "github:jitsi/lib-jitsi-meet#4baeb98964c6656b4ddb3fd5d355a9ddfa03fe74",
+    "lib-jitsi-meet": "github:saghul/lib-jitsi-meet#05cdfbe47bd4fdc6646d0e3984b31ebcb1d452ad",
```

Comment:
No, it's just so they are tested together in this particular case. It's a bit tricky because the way we build LJM changes now. The plan is to land the LJM part first, then bump it properly here.